Classes/Command/ProjectionCommandController.php

```diff
@@ -49,54 +54,90 @@ public function listCommand()
+     * and the event types which are processed by this projector.
+     *
+     * @param string $projection The projection identifier; see projection:list for possible options
      * @return void
+     * @see neos.cqrs:projection:list
      */
     public function describeCommand($projection)
     {
-        $projection = $this->projectionManager->getProjection($projection);
-        $this->outputLine('<b>PROJECTION:</b>');
-        $this->outputLine('  <i>%s</i>', [$projection->getFullIdentifier()]);
-        $this->outputLine();
-        $this->outputLine('<b>REPLAY:</b>');
-        $this->outputLine('  %s projetion:replay %s', [$this->getFlowInvocationString(), $projection->getShortIdentifier()]);
-        $this->outputLine();
-        $this->outputLine('<b>PROJECTOR:</b>');
-        $this->outputLine('  %s', [$projection->getProjectorClassName()]);
-        $this->outputLine();
+        try {
+            $projection = $this->projectionManager->getProjection($projection);
+            $this->outputLine('<b>PROJECTION:</b>');
+            $this->outputLine('  <i>%s</i>', [$projection->getFullIdentifier()]);
+            $this->outputLine();
+            $this->outputLine('<b>REPLAY:</b>');
+            $this->outputLine('  %s projection:replay %s', [$this->getFlowInvocationString(), $projection->getShortIdentifier()]);
+            $this->outputLine();
+            $this->outputLine('<b>PROJECTOR:</b>');
+            $this->outputLine('  %s', [$projection->getProjectorClassName()]);
+            $this->outputLine();
 
-        $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
-        foreach ($projection->getEventTypes() as $eventType) {
-            $this->outputLine('  * %s', [$eventType]);
+            $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
+            foreach ($projection->getEventTypes() as $eventType) {
+                $this->outputLine('  * %s', [$eventType]);
+            }
+        } catch (\Exception $e) {
+            $this->outputLine('<error>%s</error>', [$e->getMessage()]);
+            $this->quit(1);
         }
     }
 
     /**
-     * @param string $text
-     * @param int $maximumCharacters
-     * @return mixed
+     * Replay projections
+     *
+     * This command allows you to replay all relevant events for one or all projections.
+     * You can specify the special identifier "all" for replaying _all_ existing projections.
+     *
+     * @param string $projection The projection identifier; see projection:list for possible options
+     * @return void
+     * @see neos.cqrs:projection:list
      */
-    private function truncate($text, $maximumCharacters = 36)
+    public function replayCommand($projection)
     {
-        $length = strlen($text);
-        if ($length <= $maximumCharacters) {
-            return $text;
+        $eventsCount = 0;
+
+        try {
+            if ($projection === 'all') {
```

Comment:
to be honest: I prefer custom commands (like `projection:replayAll`)  over these magic strings.. Even though it's probably unlikely that someone calls their projection "all" it just makes the code less readable... and adds magic ;)